components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth/dao/OAuthAppDAO.java

```diff
@@ -1350,8 +1451,12 @@ private String getFirstPropertyValue(Map<String, List<String>> propertyMap, Stri
         return CollectionUtils.isNotEmpty(propertyMap.get(key)) ? propertyMap.get(key).get(0) : null;
     }
 
-    private boolean isOIDCAudienceEnabled() {
-        return OAuth2ServiceComponentHolder.isAudienceEnabled();
+    private boolean isOIDCIdTokenAudienceEnabled() {
+        return OAuth2ServiceComponentHolder.isIdTokenAudienceEnabled();
```

Comment:
add a new line after the method signature. check all applicable places

Reply:
Fixed with #[093969a](https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/1902/commits/093969a04c03d0d130818fe69a565ee8b85b5e1b)